src/InterestPayout.Common/Cqrs/RabbitMq/Consumer.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Linq;
+using Common.Log;
+using Lykke.Common.Log;
+using RabbitMQ.Client;
+
+namespace InterestPayout.Common.Cqrs.RabbitMq
+{
+    internal class Consumer : DefaultBasicConsumer, IDisposable
```

Comment:
it would be good to add where it was copied from. Here and for other classes

Reply:
`+`